src/main/scala/wordbots/CodeGenerator.scala

```diff
@@ -144,4 +149,44 @@ object CodeGenerator {
   private def getLabelName(label: Label): String = {
     label.getClass.getSimpleName.toLowerCase.split('$')(0)
   }
+
+  //is this a comparison like "{attribute} == {integer}"?
+  private def attributeEquivalencyComp (att: Attribute) : (Condition => Boolean) =
```

Comment:
I would move all these methods out of this PR since they're not used right now. You can bring them back in a later PR.